Tests/LibGfx/TestFontHandling.cpp

```diff
@@ -142,7 +142,7 @@ TEST_CASE(test_write_to_file)
 
     char path[] = "/tmp/new.font.XXXXXX";
     EXPECT(mkstemp(path) != -1);
-    EXPECT(!font->write_to_file(path).is_error());
+    EXPECT(!font->write_to_file(StringView { path, sizeof(path) }).is_error());
```

Comment:
This sizeof will include a null terminator

Reply:
What is this change actually for?

Reply:
This is for the `BitMapFont` -> `StringView` change, before it was creating a `DeprecatedString(char const*)`